students/grzegorz_bajorski/lesson_04_unit_testing/the_length_of_the_segment_tests.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from the_length_of_the_segment import distance
+
+
+class MyTest(pytest.TestCase):
```

Comment:
module 'pytest' has no attribute 'TestCase'. To group test cases into the class you can do just: 
```class MyTest(object)```